backend/remote/backend_common.go

```diff
@@ -284,6 +276,10 @@ func (b *Remote) costEstimate(stopCtx, cancelCtx context.Context, op *backend.Op
 			}
 		}
 
+		if b.CLI != nil {
+			b.CLI.Output("\n------------------------------------------------------------------------\n")
```

Comment:
I moved the starting output `----` here because on [line 274](https://github.com/hashicorp/terraform/pull/27735/files#diff-89e31f1ffacee792232c1f81ec9406be8e83db062a81fa2f74080098079748e8R274) we check to ensure that there are no errors/cancellations.

Reply:
Does this mean that we might see a couple of lines of `----`s if we hit `CostEstimatePending, tfe.CostEstimateQueued`? I know this kind of procedural flow makes it hard to do this correctly, but I'm thinking we shouldn't show this separator every iteration